lib/appsignal/check_in/cron.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Appsignal
+  module CheckIn
+    class Cron
+      class << self
+        # @api private
+        def transmitter
+          @transmitter ||= Appsignal::Transmitter.new(
+            "#{Appsignal.config[:logging_endpoint]}/checkins/cron/json"
```

Comment:
@matsimitsu This is the new correct endpoint, right?

Reply:
It should be `/check_ins/json` ( https://github.com/appsignal/appsignal-processor-rs/blob/a8cc1001a5acbf4174786d79414e07145de37aca/public_endpoint/http_server/src/app.rs#L68 ) 

And should look like as defined here: https://github.com/appsignal/appsignal-processor-rs/blob/develop/public_endpoint/common/src/check_in_message.rs

so, cronjob:

```json
{
   "timestamp": 123,
   "check_in_type": "Cron",
   "kind": "Start",
    "identifier": "my-cron-job",
    "digest": "cron-run-10-10-2010"
}
```

heartbeat:

```json
{
   "timestamp": 123,
   "check_in_type": "Heartbeat",
    "identifier": "my-hearbeat"
}
```

Reply:
Thanks!

Reply:
Thanks @matsimitsu, noted! Will be fixed soon.

Reply:
@matsimitsu To clarify: `identifier` is what used to be `name`, and `digest` is what used to be `id`, correct?

Reply:
@unflxw that is correct!